### PR TITLE
Adds the repos/license API endpoint

### PR DESCRIPTION
**Pull request by arall**
https://developer.github.com/v3/licenses/#get-the-contents-of-a-repositorys-license

### CODE REVIEW

**Comment by acrobat** [comment]
Thanks for the PR @arall! Can you also add a testcase for this endpoint and a docs entry? Thanks!

File: lib/Github/Api/Repo.php [hunk 1]
```diff
@@ -82,6 +82,21 @@ public function activity($username, $repository)
         return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/stats/commit_activity');
     }
 
+    /**
+     * Get the contents of the repository's license file, if one is detected.
```
Comment (dereuromark) [inline lib/Github/Api/Repo.php:86]: there is no comma before if.

Comment (acrobat) [reply lib/Github/Api/Repo.php:86]: I don't understand the comment here? But it's a exact copy of description from the github api docs, so ok for me!

Comment (dereuromark) [reply lib/Github/Api/Repo.php:86]: Just a grammar tip.

Comment (arall) [reply lib/Github/Api/Repo.php:86]: As @acrobat mentioned, I've just copied the definition from the official docs:
https://developer.github.com/v3/licenses/#get-the-contents-of-a-repositorys-license

Comment (dereuromark) [reply lib/Github/Api/Repo.php:86]: Interesting. The comma is only valid if the sentence starts with the if clause.
But then never mind :)

Comment (GrahamCampbell) [reply lib/Github/Api/Repo.php:86]: The English language doesn't really have strict rules on commas. In general, if there's a pause when you read it, you're allowed to put in a comma. :)

Comment (dereuromark) [reply lib/Github/Api/Repo.php:86]: https://www.google.com/amp/s/dictionary.cambridge.org/de/amp/britisch-grammatik/conditionals
I am talking about normal rules. Of course you can still do whatever you want^^

**Comment by GrahamCampbell** [comment]
This needs re-targetting to 2.x.

**Comment by acrobat** [comment]
Development has stalled in this PR, so I'm closing this for now. @arall if you still have time to finish it, please rebase against the current master, apply the requested changes and reopen the PR or if anyone else needs this feature please open a new PR. Thanks!

**Pull request closed without merging**